lib/scnnr/response.rb

```diff
@@ -34,8 +34,12 @@ def handle_recognition(recognition)
       return recognition unless recognition.error?
 
       case recognition.error['type']
+      when 'internal-server-error'
+        raise UnexpectedError, @response
       when 'unexpected-content', 'bad-request'
         raise RequestFailed, recognition.error
+      when 'download-timeout', 'invalid-image'
+        raise RecognitionFailed, recognition
       else raise RecognitionFailed, recognition
```

Comment:
I think:
`UnexpectedError`: For all unknown errors
`RecognitionFailed`: For situations when recognition was failed

So `RecognitionFailed` basically has more concrete information.
After checking all error types (in `else` clause`), should we return `RecognitionFailed`?

Reply:
> UnexpectedError: For all unknown errors

But also 'internal-server-errors', right?
I've updated it here -> https://github.com/NEWROPE/scnnr-ruby/pull/25/commits/b0f42a9afbb3f832f6f0284644083a328934a18b

Reply:
Sorry, as we discussed in person, `internal-server-error` means the provider's website which provides a requested image dies, so RecognitionFailed was correct...
I also updated the documentation on the story.